lib/clearance/password_strategies/bcrypt_migration_from_sha1.rb

```diff
@@ -1,6 +1,13 @@
 module Clearance
   module PasswordStrategies
     module BCryptMigrationFromSHA1
+      DEPRECATION_MESSAGE = "[DEPRECATION] The BCryptMigrationFromSha1 " \
```

Comment:
I don't know if there are any other warnings in this project, but would it be a good idea to use a heredoc and wrap this at 72 or 80 characters?